src/systemctl/systemctl-list-dependencies.c

```diff
@@ -146,7 +146,7 @@ static int list_dependencies_one(
 
         if (circular && !arg_plain) {
                 printf("  ");
-                r = list_dependencies_print("...", level, branches, /* last = */ true);
+                r = list_dependencies_print("...", _UNIT_ACTIVE_STATE_INVALID, level, branches, /* last = */ true);
```

Comment:
This will now be printed in red. On purpose?

Reply:
It won't. This is covered by
```c
if (arg_plain || state == _UNIT_ACTIVE_STATE_INVALID)
        printf("  ");
```
in `list_dependencies_print`.

Reply:
Ah, OK.